ddtrace/contrib/pytest/plugin.py

```diff
@@ -102,6 +104,12 @@ def pytest_runtest_protocol(item, nextitem):
         span.set_tag(test.SUITE, item.module.__name__)
         span.set_tag(test.TYPE, SpanTypes.TEST.value)
 
+        # Parameterized test cases will have a `callspec` attribute attached to the pytest Item object.
+        # Pytest docs: https://docs.pytest.org/en/6.2.x/reference.html#pytest.Function
+        if getattr(item, "callspec", None):
+            params = {"parameters": item.callspec.params, "metadata": {}}
+            span.set_tag(test.PARAMETERS, json.dumps(params, default=lambda x: x.__dict__ if x.__dict__ else str(x)))
```

Comment:
we should probably extract this json dumping into a helper function that we can write tests for, that way we can use hypothesis maybe to test as many possible scenarios as possible.